elyra/pipeline/component_registry.py

```diff
@@ -202,13 +236,15 @@ class CachedComponentRegistry(ComponentRegistry):
 
     _cache: List[Component] = list()
```

Comment:
With the introduction of `_categories` I feel like `_cache` should be named something like `_cached_components` along side `_cached_categories`.